app/controllers/concerns/application_shared.rb

```diff
@@ -140,4 +140,11 @@ def find_session_taxonomy(taxonomy, user)
     end
     determined_taxonomy
   end
+
+  def find_selected_columns
+    instance_variable_set(
```

Comment:
What is the context in this context? Where does the instance variable get set? On the instance of the controller?

Reply:
Yes, this should work as `find_resource`: callback that is called before `index`, so we have `@selected_columns` for that controller/table, which then can be used in `index.html.erb` implicitly by `render_selected_column_[ths|tds]`

Reply:
Any reason why we can't just `@selected_columns = Foreman::SelectableColumns::Storage.selected_by(User.current, controller_name)`?

Reply:
Nope, changed :)